server/graphql/v1/mutations/orders.js

```diff
@@ -946,6 +947,12 @@ export async function addFundsToCollective(order, remoteUser) {
     orderData.data.platformFeePercent = order.platformFeePercent;
   }
 
+  if (!isNil(order.platformTipPercent)) {
```

Comment:
We don't do it for `hostFeePercent` and `platformFeePercent`, but it would actually be nice to make sure this number is between 0 and 100 (I'm not sure if we verify that later in the process).

Reply:
@Betree : Thanks. 👍🏽 . Done, I've added the validations for `hostFeePercent` and `platformFeePercent` as well. 😄